src/GeneratedInjectorDelegator.php

```diff
@@ -22,7 +22,9 @@ public function __invoke(ContainerInterface $container, $name, callable $callbac
     {
         $config = $container->has('config') ? $container->get('config') : [];
         $aotConfig = $config['dependencies']['auto']['aot'] ?? [];
-        $namespace = empty($aotConfig['namespace']) ? 'Laminas\Di\Generated' : $aotConfig['namespace'];
+        $namespace = ! isset($aotConfig['namespace']) || $aotConfig['namespace'] === ''
```

Comment:
An empty array is a result of a potential misconfiguration. The `empty()` check will swallow this and not inform the user about it. Using the null coalescing check would not catch an empty string. Therefore `isset()` and explicit empty string check.